docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -21,8 +21,8 @@ What is a tool?
 A tool represents a single command line tool wrapped in a descriptor language.  Languages that formally describe tools (such as CWL) may chain them together into a workflow.
 
 
-What is a service?
-------------------
+Introduction to Services
+==============================================
 
-Services are meant to be long running processes, usually web services or interactive applications, that can be launched by a user. Dockstore services are currently in beta. 
+Services are containerized programs that are meant to be long running processes, usually web services or interactive applications, and that can be launched by a user into different environments. Dockstore services are currently in beta. 
```

Comment:
@coverbeck Are services containerized? I presume so, but since we don't have a way to search for services I'm unsure.